libs/gi/ui/src/components/team/TeamCard.tsx

```diff
@@ -68,17 +73,42 @@ export function TeamCard({
           flexDirection: 'column',
         }}
       >
-        <CardActionArea onClick={() => onClick()} sx={{ p: 1 }}>
-          <Typography sx={{ display: 'flex', gap: 1 }} variant="h6">
-            <span>{name}</span>{' '}
-            {description && (
-              <BootstrapTooltip title={<Typography>{description}</Typography>}>
-                <InfoIcon />
-              </BootstrapTooltip>
-            )}
-          </Typography>
-        </CardActionArea>
-
+        <Box
+          sx={{
+            display: 'flex',
+            flexDirection: 'row',
+          }}
+        >
+          <CardActionArea onClick={() => onClick()} sx={{ p: 1 }}>
+            <Typography sx={{ display: 'flex', gap: 1 }} variant="h6">
+              <span>{name}</span>{' '}
+              {description && (
+                <BootstrapTooltip
+                  title={<Typography>{description}</Typography>}
+                >
+                  <InfoIcon />
+                </BootstrapTooltip>
+              )}
+            </Typography>
+          </CardActionArea>
+          <TeamDelModal
+            teamId={teamId}
+            show={showDel}
+            onHide={onHideDel}
+            onDel={function (): void {}}
+          />
+          <Button
```

Comment:
I would advice using [IconButton](https://mui.com/material-ui/react-button/#icon-button) and keep it as a red icon on the top right.

Reply:
![Screenshot 2024-05-16 at 11 57 39 PM](https://github.com/frzyc/genshin-optimizer/assets/23622319/9ebddee3-548b-4c0d-8f04-6de938421377)

something like this look good?

Reply:
Perfect, exactly what I wanted :D